deployments/notifyRewardAmount/optimism/givpower_distribute_extended.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-use-before-define */
+const hre = require("hardhat");
+const { sendReportEmail } = require("../../mailService/mailService");
+const { ethers } = hre;
+
+const pools = [
+    {
+        address: "0xD93d3bDBa18ebcB3317a57119ea44ed2Cf41C2F2",
```

Comment:
```suggestion
        address: "0x301C739CF6bfb6B47A74878BdEB13f92F13Ae5E7",
```

according to https://www.notion.so/giveth/Deployment-52935e6579034f1b940bbef161e82229

Reply:
@aminlatifi Nice catch, thank you